src/Servers/Kestrel/Core/src/Internal/ConfigurationReader.cs

```diff
@@ -20,6 +20,7 @@ internal class ConfigurationReader
         private const string EndpointsKey = "Endpoints";
         private const string UrlKey = "Url";
         private const string ClientCertificateModeKey = "ClientCertificateMode";
+        private const string SNIKey = "SNI";
```

Comment:
bikeshedding: Sni? Naming guidelines is acronym of 3 or more letters are camelcased, e.g. Url, Ssl, Tls.